release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -527,6 +547,15 @@ You can configure PTP ordinary clocks with added redundancy on `x86` architectur
 
 For more information, see xref:../networking/ptp/about-ptp.adoc#ptp-dual-ports-oc_about-ptp[Using dual-port NICs to improve redundancy for PTP ordinary clocks].
 
+[id="ocp-4-19-networking-conditional-webhook_{context}"]
+==== Support for conditional webhook matching in the SR-IOV Network Operator
+
+You can now enable the `featureGates.resourceInjectorMatchCondition` feature in the `SriovOperatorConfig` object to limit the scope of the Network Resources Injector webhook. If this feature is enabled, the webhook applies only to pods with the secondary network annotation `k8s.v1.cni.cncf.io/networks`.
+
+If you set the `failurePolicy` field to `Fail` after enabling the `resourceInjectorMatchCondition` feature, the webhook applies only to pods with the secondary network annotation `k8s.v1.cni.cncf.io/networks`. If the webhook is unavailable, pods without this annotation are still deployed, preventing unnecessary disruptions to cluster operations.
```

Comment:
This part is not right.

The user doesn't configure the `failurePolicy` that is done by the operator when the feature gate is enabled.
I think here we should add something like

without this feature the failurePolicy for the resource injector is ingnore and that can load to pods requesting sriov networks not having the resource injected. + the improvement here is "If the webhook is unavailable, pods without this annotation are still deployed, preventing unnecessary disruptions to cluster operations."